Source/Scene/GlobeSurfaceTileProvider.js

```diff
@@ -508,8 +579,13 @@ define([
      * @exception {DeveloperError} <code>loadTile</code> must not be called before the tile provider is ready.
      */
     GlobeSurfaceTileProvider.prototype.loadTile = function(frameState, tile) {
+        if (stopLoad) {
+            return;
+        }
         GlobeSurfaceTile.processStateMachine(tile, frameState, this._terrainProvider, this._imageryLayers, this._vertexArraysToDestroy);
         var tileLoadedEvent = this._tileLoadedEvent;
+
+        // TODO: creating a new function for every loaded tile every frame?!
```

Comment:
Please please please do not.  😢 

I went to great pains to avoid this in the early 3D Tiles engine.

Reply:
This creation of a new function every frame is already in master. I was surprised to see it and left this TODO to remind myself to maybe do something about it.